pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -51,6 +51,11 @@ func (a Adapter) Push(path string, ignoredFiles []string, forceBuild bool, globE
 		return errors.Wrap(err, "unable to create or update component")
 	}
 
+	_, err = a.Client.WaitForDeploymetRollout(a.ComponentName)
+	if err != nil {
+		return err
```

Comment:
Should we wrap this error like we do for the other errors we return from this function?

Reply:
fixed